src/sentry/api/endpoints/internal/integration_proxy.py

```diff
@@ -81,11 +81,11 @@ def _validate_request(self, request: HttpRequest) -> bool:
         from sentry.shared_integrations.client.proxy import IntegrationProxyClient
 
         # Get the organization integration
-        org_id_header = request.headers.get(PROXY_OI_HEADER)
-        if org_id_header is None or not org_id_header.isnumeric():
+        org_integration_id_header = request.headers.get(PROXY_OI_HEADER)
+        if org_integration_id_header is None or not org_integration_id_header.isnumeric():
             logger.info("integration_proxy.missing_org_integration", extra=self.log_extra)
             return False
-        org_integration_id = int(org_id_header)
+        org_integration_id = int(org_integration_id_header)
```

Comment:
though this was the bug but it's just a naming change, though it could have made a bug later on 🤷